app/components/HeaderComp.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import AddToCart from "./AddToCart"
+import { HEADER_LABEL_MESSAGE } from '../constant';
+import logo from '../images/logo.png'
+import cart from '../images/cart.svg'
+
+const HeaderComp = (props) => {
+    const { cartItem, handleCount } = props;
+    const [toggleCart, settoggleCart] = useState(false)
+    const openAddtocart = (flag) => {
+        settoggleCart(flag)
+    }
+    return (
+        <header className="header">
+            <div className="inner-header floatcontainer">
+                <div className="logo-container">
+                    <img src={logo} alt="App Logo" />
```

Comment:
alt tags provide meaningful info through screen readers. Add small but meaningful info in alt. Just "App Logo" doesnt help much

Reply:
Done